### PR TITLE
Add tests without modifying code

### PR DESCRIPTION
**Pull request by gio-puter**
### Describe your change:
Contributes to #9943
Added doctest to maths/largest_of_very_large_numbers.py
Added doctest to strings/words_patterns.py
Added doctest to ciphers/onepad_cipher.py

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Review by imSanko — COMMENTED** [review]
Use Different PR !! one PR can't change the entire stuff, so it should be one by one which becomes easy for the maintainers

File: ciphers/onepad_cipher.py [hunk 1]
```diff
@@ -4,7 +4,17 @@
...
+        """
+        Function to encrypt text using pseudo-random numbers
+        >>> Onepad().encrypt("")
+        ([], [])
+        >>> random.seed(1)
+        >>> Onepad().encrypt(" ")
+        ([6969], [69])
+        >>> random.seed(1)
+        >>> Onepad().encrypt("Hello")
+        ([9729, 114756, 4653, 31309, 10492], [69, 292, 33, 131, 61])
```
Comment (cclauss) [inline ciphers/onepad_cipher.py:16]: Let's make it raise some Exceptions:
```suggestion
        ([9729, 114756, 4653, 31309, 10492], [69, 292, 33, 131, 61])
        >>> Onepad().encrypt(1)
        >>> Onepad().encrypt(1.1)
        >>> Onepad().encrypt([])
```

File: ciphers/onepad_cipher.py [hunk 2]
```diff
@@ -17,7 +27,21 @@ def encrypt(text: str) -> tuple[list[int], list[int]]:
...
+        Function to decrypt text using pseudo-random numbers.
+        >>> Onepad().decrypt([], [])
+        ''
+        >>> Onepad().decrypt([35], [])
+        ''
+        >>> Onepad().decrypt([], [35])
+        Traceback (most recent call last):
+        ...
+        IndexError: list index out of range
+
```
Comment (cclauss) [inline ciphers/onepad_cipher.py:40]:
```suggestion
```

File: maths/largest_of_very_large_numbers.py [hunk 3]
```diff
@@ -4,6 +4,23 @@
...
+    0
+
+    >>> res(3, 0)
+    1
+
+    >>> res(-1, 5)
+    Traceback (most recent call last):
+    ...
+    ValueError: math domain error
+
```
Comment (cclauss) [inline maths/largest_of_very_large_numbers.py:22]: It is better if the tests do not require extra scrolling when someone is reading the code.  If they want to read all the tests, they can take the time to do so but often readers will want to look at the function signature and then jump straight to the code.  Keeping the tests in tight block will allow them them to study the function signature and the code without too much visual distraction.
```suggestion
    >>> res(0, 5)
    0
    >>> res(3, 0)
    1
    >>> res(-1, 5)
    Traceback (most recent call last):
    ...
    ValueError: math domain error
```

File: strings/word_patterns.py [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
     """
+    Returns numerical pattern of character appearances in given word
+    >>> get_word_pattern()
+    Traceback (most recent call last):
+    ...
+    TypeError: get_word_pattern() missing 1 required positional argument: 'word'
+    >>> get_word_pattern("")
+    ''
+    >>> get_word_pattern(" ")
+    '0'
```
Comment (cclauss) [inline strings/word_patterns.py:11]: Please put the tests that work correctly first and then afterward the ones that raise exceptions.

Also, please add:
        >>> get_word_pattern(1)
        >>> get_word_pattern(1.1)
        >>> get_word_pattern([])

**Review by cclauss — CHANGES_REQUESTED** [review]
Nice!  Just a few changes...

**Comment by cclauss** [comment]
@imSanko You are correct but in this case, **_adding tests without changing code_** makes this one easy to review.

I can read through the tests and see if they make sense and hit some corner cases and then I can look at the `build` output and see that they are running and none of these files is among our ten slowest running tests and then I can approve.

**Comment by gio-puter** [comment]
Ok. Awesome. I went ahead and made the requested format changes and additions.

**Review by cclauss — APPROVED** [review]
Nice!!

**Pull request merged**